### PR TITLE
feat(profile): allow users to view other user's profiles

### PR DESCRIPTION
**Pull request by irtiza-rizvi**
AMLOS-753

**Description**
As a user I want to view another user's profile to view their contact info, org, avatar, and possibly applications they own.

**Acceptance Criteria**
A user can go to a link (through a listing, or directly) that will display the profile info of another user

**How To Test**
Pull this branch
Pull `add_avatar_fields` from `ozp-backend` if https://github.com/aml-development/ozp-backend/pull/444 is not merged

On a listing page, you should see links under _Owners_, _Reviews_, and _Listing Administration --> Listing Changes_ to user profiles.  Clicking a link should direct you to the user's profile page.

All profile links should show the user's avatar icon, or the default user icon if the user does not have an avatar.

Changing the URL manually should load the correct pages, ie:
`http://localhost:8080/#/profile/13` --> jones's profile page
`http://localhost:8080/#/profile` --> logged in user's profile page
`http://localhost:8080/#/profile/edit` --> logged in user's edit profile page

### CODE REVIEW

File: src/services/profile.js [hunk 1]
```diff
@@ -7,7 +7,15 @@ export function searchProfile(item) {
     )
 }
 
-export function getProfile() {
+export function getProfile(id) {
+    return $.ajax({
+        url: `${API_URL}/api/profile/${id}`,
```
Comment (blynch11p) [inline src/services/profile.js]: that darned trailing `/` again...   this call fails in FF.

Comment (irtiza-rizvi) [reply src/services/profile.js]: Awesome.  Pushed a fix!

**Comment by blynch11p** [comment]
Shows profile of the user, including their avatar. :+1:

**Comment by flesher1** [comment]
:+1: Looks good. You have one console warning in the listing overview section though. Its creating a list and links need keys or something

**Pull request merged**